### ISSUE TITLE
Add dark theme

### ISSUE THREAD

**Issue by vitaly-zdanevich** [brandur/sorg#272]
Through https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

**Comment by brandur** [comment]
Yeah, I'll do this at some point probably. My CSS is a bit of a mess right now though, so it'll probably have to wait until I do a redesign.
